apps/shell/tash_command.c

```diff
@@ -564,9 +564,9 @@ int tash_execute_cmd(char **args, int argc)
 			if (tash_cmds_info.cmd[cmd_idx].exec_type == TASH_EXECMD_SYNC) {
 				/* function call to execute SYNC command */
 #ifdef CONFIG_PM
-				pmdrv_fd = open("/dev/pm", O_WRONLY);
+				pmdrv_fd = open(PM_DRVPATH, O_WRONLY);
```

Comment:
Do we need to open the pm driver here as well?

Reply:
Yes, to suspend `TASH` domain